adafruit_tsl2561.py

```diff
@@ -247,13 +253,13 @@ def _compute_lux(self):
         lux *= _TIME_SCALE[self.integration_time]
         return lux
 
-    def _enable(self):
+    def _enable(self) -> None:
         self._write_control_register(_CONTROL_POWERON)
 
-    def _disable(self):
+    def _disable(self) -> None:
         self._write_control_register(_CONTROL_POWEROFF)
 
-    def _read_register(self, reg, count=1):
+    def _read_register(self, reg: int, count: int = 1) -> Union[int, Tuple[int, int]]:
```

Comment:
So technically, this **can** return `None`, but only if used improperly.  That `return None` at the end of the function should probably be change to an exception.  Could you change the function to be:

```python
    def _read_register(self, reg: int, count: int = 1) -> Tuple[int, Tuple[int, int]]:
        if count not in (1, 2):
            raise RuntimeError("Can only read up to 2 consecutive registers")
        self.buf[0] = _COMMAND_BIT | reg
        if count == 2:
            self.buf[0] |= _WORD_BIT
        with self.i2c_device as i2c:
            i2c.write_then_readinto(self.buf, self.buf, out_end=1, in_start=1)
        if count == 1:
            return self.buf[1]
        return self.buf[1], self.buf[2]
```